unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/FriendRequestHUD/SentFriendRequestHUDController.cs

```diff
@@ -100,19 +102,27 @@ private async UniTask CancelAsync(CancellationToken cancellationToken = default)
 
             try
             {
-                await friendsController.CancelRequestAsync(friendRequestId)
-                                       .Timeout(TimeSpan.FromSeconds(10));
+                FriendRequest request = await friendsController.CancelRequestAsync(friendRequestId)
+                                                                     .Timeout(TimeSpan.FromSeconds(10));
```

Comment:
Let's create a constant for the 10 seconds so that it's easier to spot it and change it in the future if required

Reply:
done